### PR TITLE
Feature/65 add accesskeys status toggle

### PR DESCRIPTION
**Pull request by AsmaaELM**
Issue: https://scality.atlassian.net/browse/ZKUI-65

### CODE REVIEW

File: src/react/account/AccountUserAccessKeys.jsx [hunk 1]
```diff
@@ -43,6 +48,41 @@ const CreatedOnCell = rowValue => {
   );
 };
 
+const ToggleAccessKeyStatus = rowValue => {
+  const { accessKey, status: accessKeyStatus } = rowValue;
+  const IAMClient = useIAMClient();
+  const { IAMUserName } = useParams();
+  const loading = useSelector(
+    (state: AppState) => state.networkActivity.counter > 0,
+  );
```
Comment (JBWatenbergScality) [inline src/react/account/AccountUserAccessKeys.jsx]: I'm not sure the loading status is useful here

File: src/react/account/AccountUserAccessKeys.jsx [hunk 2]
```diff
@@ -43,6 +48,41 @@ const CreatedOnCell = rowValue => {
...
+    {
+      onSuccess: () =>
+        queryClient.invalidateQueries(['listIAMUserAccessKey', IAMUserName]),
+    },
+  );
+
+  return (
+    <div>
+      <Toggle
+        disabled={loading}
```
Comment (JBWatenbergScality) [inline src/react/account/AccountUserAccessKeys.jsx]:
```suggestion
        disabled={IAMClient === null}
```

File: src/react/actions/user.js [hunk 3]
```diff
@@ -144,6 +144,21 @@ export function deleteAccessKey(accessKey, userName) {
...
+      .updateAccessKey(accessKey, status, userName)
+      .then(() => {
+        dispatch(listAccessKeys(userName));
+      })
+      .catch(error => dispatch(handleClientError(error)))
+      .catch(error => dispatch(handleApiError(error, 'byModal')))
+      .finally(() => dispatch(networkEnd()));
+  };
+}
+
```
Comment (JBWatenbergScality) [inline src/react/actions/user.js]: This code is never used, it can be removed

File: src/react/account/AccountUserAccessKeys.jsx (hunk 1 above)
Comment (AsmaaELM) [reply src/react/account/AccountUserAccessKeys.jsx]: Loading deleted -> done

File: src/react/actions/user.js (hunk 3 above)
Comment (AsmaaELM) [reply src/react/actions/user.js]: Code deleted -> done

**Pull request closed without merging**